extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.tsx

```diff
@@ -242,6 +256,25 @@ function TrackedCornerstoneViewport(props) {
             />
           )}
         </div>
+        {isCineEnabled && (
```

Comment:
we need to add this to the non-tracked (basic) viewport as well. I guess if you put it there, then we don't it here, since we use `getCornerstoneViewport`